applications/app/controllers/CrosswordsController.scala

```diff
@@ -54,6 +58,83 @@ object CrosswordsController extends Controller with ExecutionContexts {
   }
 }
 
+object CrosswordSearchController extends Controller with ExecutionContexts {
+  def searchForm() = Action { implicit request =>
+    Ok(views.html.crosswordSearch(CrosswordSearchPage))
```

Comment:
If no caching then needs explicit "do not cache" headers - we have a `NoCache(...)` helper.

Leaving off the headers means the internet can do what it wants with it...